### PR TITLE
Webhook processor for sending to Jenkins  TE-2069

### PR DESCRIPTION
**Pull request by jzoldak**

### CODE REVIEW

File: lambdas/webhook_processor/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Webhook Processor
+
+Processes webhooks by reading them from a kinesis stream and forwarding to Jenkins
+
+To zip and upload a new version, using the aws cli:
+```
+zip webhook_processor.zip webhook_processor.py
+aws s3 cp webhook_processor.zip s3://edx-testeng-lambda-webhook/
```
Comment (estute) [inline lambdas/webhook_processor/README.md]: nit: 's3://edx-testeng-lambda-webhooks/' ?

File: lambdas/webhook_processor/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Webhook Processor
+
+Processes webhooks by reading them from a kinesis stream and forwarding to Jenkins
```
Comment (estute) [inline lambdas/webhook_processor/README.md]: Perhaps we should explain a little about the purpose of this within the whole 'interceptor' design.

File: lambdas/webhook_processor/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Unfortunately finding the version number is not possbile via the aws cli,
+only through the REST API, the SDKs, or the console.
+Go to the s3 bucket web interface (https://console.aws.amazon.com/s3)
+Navigate to the bucket, and then the .zip file.
+Look for the "Version ID" on the Overview tab.
+
+This is what you will put in the terraform.tfvars file.
+  lambda2_version = "<the new version id>"
+
+Note the target (jenkins) URL is also configured in the terraform.tfvars file.
```
Comment (estute) [inline lambdas/webhook_processor/README.md]: How do we plan on modifying the target in the future? Will we need to reapply the terraform each time or will it be a available via an environment variable?

File: lambdas/webhook_processor/webhook_processor.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+from botocore.vendored import requests
+
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logger.info('Loading function')
+
+def lambda_handler(event, context):
+
+    url = os.environ.get('TARGET_URL')
```
Comment (estute) [inline lambdas/webhook_processor/webhook_processor.py]: There are going to be two endpoints we care about:
* $url/ghprbhook
* $url/github-webhook (for webhooks sent when merging to master)

**Comment by jzoldak** [comment]
Closing in favor of #91

**Pull request closed without merging**